lib/workflow/messengers/messenger-AMQP.js

```diff
@@ -0,0 +1,76 @@
+// Copyright 2015, EMC, Inc.
+
+'use strict';
+
+module.exports = amqpMessengerFactory;
+amqpMessengerFactory.$provide = 'Task.Messengers.AMQP';
+amqpMessengerFactory.$inject = [
+    'Protocol.Task',
+    'Protocol.Events',
+    'Protocol.TaskGraphRunner',
+    'Promise'
+];
+function amqpMessengerFactory(
+    taskProtocol,
+    eventsProtocol,
+    taskGraphRunnerProtocol,
+    Promise
+) {
+    var exports = {};
+
+    exports.subscribeRunTask = function(domain, callback) {
+        return taskProtocol.subscribeRun(domain, callback);
+    };
+
+    exports.publishRunTask = function(domain, taskId, graphId) {
+        return taskProtocol.run(domain, { taskId: taskId, graphId: graphId });
+    };
+
+    exports.subscribeCancel = function(callback) {
```

Comment:
It could, the task protocol naming was this way for long lost reasons, so we were shooting for consistency in the short term.